.github/workflows/CI.yaml

```diff
@@ -80,11 +80,13 @@ jobs:
           registry: ghcr.io
           username: ${{ secrets.DATALAND_SKYMINDERCLIENT_USER }}
           password: ${{ secrets.DATALAND_SKYMINDERCLIENT_TOKEN }}
+      - name: Retrieve SSL certificates for dataland-local.duckdns.org
+        run: mkdir -p ~/.ssh/ && echo "${{ secrets.DATALAND_LETSENCRYPT_HOST_KEYS }}" >>  ~/.ssh/known_hosts && echo "${{ secrets.SSH_PRIVATE_KEY }}" > ~/.ssh/id_rsa && chmod 600 ~/.ssh/id_rsa && mkdir -p ./local/certs && scp ubuntu@dataland-letsencrypt.duckdns.org:/etc/letsencrypt/live/dataland-local.duckdns.org/* ./local/certs
```

Comment:
ich finde Zeile 84 hat verdient in ein eigenes Script zu wandern :-)

Reply:
Kann ich mich mit anfreunden

Reply:
ok